fuuk/people/admin/options.py

```diff
@@ -104,6 +104,8 @@ class AttachmentInlineAdmin(TabularInline):
 class CourseAdmin(TranslationAdmin):
     list_display = ('name', 'ls', 'zs', 'code')
     ordering = ('code', )
+    fields = ['code'] + get_translation_fields('name') + ['lectors', 'practical_lectors', 'ls', 'zs'] + \
+        get_translation_fields('annotation') + get_translation_fields('note')
```

Comment:
`modeltranslation` correctly expands field names into translated fields in `fields` attribute. It only doesn't work for `search_fields`. So the `get_translation_fields` is excessive here.